en/tutorials-and-examples/cms/authorization.rst

```diff
@@ -78,7 +78,15 @@ The Authorization plugin models authorization and permissions as Policy classes.
 These classes implement the logic to check whether or not a **identity** is
 allowed to **perform an action** on a given **resource**. Our **identity** is
 going to be our logged in user, and our **resources** are our ORM entities and
-queries. Lets use bake to generate a basic policy:
+queries. Lets use bake to generate a basic policy. In **src/Application.php** 
+add the following in the ``bootstrap()`` method::
+
+.. code-block:: bash
```

Comment:
You can remove this line. The sentence above ends in :: which enables php formatting.